### PR TITLE
Fixes #4, #5, and #6

### LINKED ISSUE #4: Add configuration for a default category
If a default location is specified, the script might randomly select a place that isn't meant for food, like home depot.

Need to add a configuration that allows a category to be added along with any other categories that the user specifies.

### PR DESCRIPTION
**Pull request by jvarness**
Or at least I'm pretty sure it does.

### CODE REVIEW

**Comment by jvarness** [comment]
@pcarn pls I need your +1 help
Also @drkenyan if you please

**Comment by drtootsie** [comment]
Do you have a code review?

**Comment by jvarness** [comment]
This is the review

File: src/lunchtime.coffee [hunk 1]
```diff
@@ -52,16 +54,19 @@ queryYelp = (msg, usrLocation, category) ->
...
-    msg.send randomBusiness.url 
+    if response.businesses.length > 0
+      msg.send 'Give this place a shot:'
+      randomBusiness = msg.random response.businesses
+      msg.send randomBusiness.name 
+      msg.send 'Yelp rating: ' + randomBusiness.rating
+      msg.send 'Total reviews: ' + randomBusiness.review_count
+      msg.send randomBusiness.url
+    else
+      msg.send "Couldn't find a place to eat there. Try again.ß"
```
Comment (drtootsie) [inline src/lunchtime.coffee]: What's with the weird character?

**Comment by drtootsie** [comment]
+1 if you can explain that odd character

File: src/lunchtime.coffee (hunk 1 above)
Comment (jvarness) [inline src/lunchtime.coffee]: Total accident. I fix it

**Pull request merged**